src/components/PlantingSites/PlantingSiteCreate.tsx

```diff
@@ -1,10 +1,150 @@
+import useForm from 'src/utils/useForm';
+import { PlantingSitePostRequestBody, postPlantingSite } from 'src/api/tracking/tracking';
+import { APP_PATHS } from 'src/constants';
+import { Link, useHistory } from 'react-router-dom';
+import useSnackbar from 'src/utils/useSnackbar';
+import TextField from '@terraware/web-components/components/Textfield/Textfield';
+import { useState } from 'react';
+import PageSnackbar from '../PageSnackbar';
+import { makeStyles } from '@mui/styles';
 
 type CreatePlantingSiteProps = {
   organization: ServerOrganization;
 };
 
+const useStyles = makeStyles(() => ({
+  backIcon: {
+    fill: theme.palette.TwClrIcnBrand,
+    marginRight: theme.spacing(1),
+  },
+  back: {
+    display: 'flex',
+    textDecoration: 'none',
+    color: theme.palette.TwClrTxtBrand,
+    fontSize: '20px',
+    alignItems: 'center',
+  },
+}));
+
 export default function CreatePlantingSite(props: CreatePlantingSiteProps): JSX.Element {
-  return <TfMain />;
+  const { isMobile } = useDeviceInfo();
+  const { organization } = props;
+  const history = useHistory();
+  const snackbar = useSnackbar();
+  const [nameError, setNameError] = useState('');
+  const classes = useStyles();
+
+  const defaultPlantingSite = (): PlantingSitePostRequestBody => ({
+    name: '',
+    organizationId: organization.id,
+  });
+
+  const [record, , onChange] = useForm<PlantingSitePostRequestBody>(defaultPlantingSite());
+
+  const goToPlantingSites = () => {
+    const plantingSitesLocation = {
+      pathname: APP_PATHS.PLANTING_SITES,
+    };
+    history.push(plantingSitesLocation);
+  };
+
+  const savePlantingSite = async () => {
+    if (!record.name) {
+      setNameError(strings.REQUIRED_FIELD);
+      return;
+    }
+    const response = await postPlantingSite(record);
+
+    if (response.requestSucceeded) {
+      snackbar.toastSuccess(strings.CHANGES_SAVED);
+      goToPlantingSites();
+    } else {
+      snackbar.toastError();
+    }
+  };
+
+  const gridSize = () => {
+    if (isMobile) {
+      return 12;
+    }
+    return 4;
+  };
+
+  return (
+    <TfMain>
+      <Container maxWidth={false}>
+        <Grid container spacing={3}>
+          <Grid item xs={12}>
+            <Link id='back' to={APP_PATHS.PLANTING_SITES} className={classes.back}>
+              <Icon name='caretLeft' className={classes.backIcon} />
+              {strings.PLANTING_SITES}
+            </Link>
+          </Grid>
+          <Grid item xs={12}>
+            <Typography fontSize='24px' fontWeight={600} margin={theme.spacing(1, 0)}>
+              {strings.ADD_PLANTING_SITE}
+            </Typography>
+            <Typography fontSize='14px' fontWeight={400} margin={theme.spacing(0, 0, 3, 0)}>
+              {strings.ADD_PLANTING_SITE_DESCRIPTION}
+            </Typography>
+          </Grid>
+          <PageSnackbar />
+          <Grid item xs={gridSize()}>
+            <TextField
+              id='name'
+              label={strings.NAME}
```

Comment:
I understand Name and Description don't have the required '*' next to them - but feels like they should? 
We could check with @chudo76 in figma.

Reply:
Name is required according to the api, and description is optional. I'm validating name when saving

Reply:
PRD says name alone is required - which is different from our other pages.
We should at least ask Chudo if we should add the '*' next to 'Name' label for consistency, yes?